package.json

```diff
@@ -123,11 +123,11 @@
     "npm-run-all": "^4.0.2",
     "qunitjs": "^2.3.2",
     "rimraf": "^2.6.1",
-    "rollup": "^0.41.6",
+    "rollup": "^0.51.8",
     "rollup-plugin-babel": "^2.7.1",
     "rollup-plugin-commonjs": "^8.0.2",
     "rollup-plugin-json": "^2.1.1",
-    "rollup-plugin-multi-entry": "^2.0.1",
+    "rollup-plugin-multi-entry": "^2.0.2",
```

Comment:
Was there any specific reason for this change?

Reply:
I believe that patch version is needed [for the update in multi-entry](https://github.com/rollup/rollup-plugin-multi-entry/issues/24#issuecomment-342229523)

Though it's possible that 2.0.1 also contains it, I'm not actually sure...